handlers.js

```diff
@@ -76,43 +75,52 @@ module.exports.root = ({ headers, cf }) =>
   respondJSON({
     payload: {
       cli: 'pip install -U print-env --pre',
-      'Web UI': 'https://portunus.netlify.app',
+      'Web UI': 'https://portunus.locus.engineering',
```

Comment:
Can we make this configurable through env var?

Reply:
done

Reply:
It will be better to put the value of this variable in `' '` while putting in gh s.ecrets

Reply:
Done!